.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,29 @@ Addresses issue #
 - [ ] My code has proper inline documentation.
 - [ ] I have added a QA Brief on the issue linked above.
 - [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).
+
+## Code Review Checklist
```

Comment:
We should probably visually separate the two new areas more from the rest of the template, almost like we have in the issue template for the workflow sections. It needs to be clear that the person opening a PR doesn't need to do anything here (and also shouldn't remove them).

Also, the original checklist above (which the PR author _should_ use) is now still only called "Checklist", we may want to clarify that, now that there are other checklists.

Reply:
Yep, updated.